pkg/api/install/install.go

```diff
@@ -34,35 +34,55 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
-
 const importPrefix = "k8s.io/kubernetes/pkg/api"
 
 var accessor = meta.NewAccessor()
 
+// availableVersions lists all known external versions for this group from most preferred to least preferred
+var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion}
```

Comment:
> The order used to be populated from the registered package, which is populated from the KUBE_API_VERSIONS environment variable, so user can change the order at runtime.
> 
> I guess you will have followup PR for register.go. Could you share your plan?

Right now it still is (note the `for` loop below), but I'd like to avoid the centralization of every external group,version tuple in existence here: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/registered/registered.go#L41 to populate the default.  Instead, I'd have a default of "everything installed based on the default specified in each install package".

I do think that this pull can also standalone (without any followup), because it makes the group registration a more forward flow.  Figure out the inputs, build your object, register.  Instead of the reverse.